src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -236,9 +236,9 @@ public function getCookies(string $format = self::COOKIES_FLAT): array
      *
      * @return void
      */
-    public function clearCookie(string $name, ?string $path = '/', ?string $domain = null, bool $secure = false, bool $httpOnly = true, ?string $sameSite = null)
+    public function clearCookie(string $name, ?string $path = '/', ?string $domain = null, bool $secure = false, bool $httpOnly = true, ?string $sameSite = null, bool $partitioned = false)
```

Comment:
This is a BC break; the parameter needs to be virtual only at first by making it commented out and adding an`@param` phpdoc annotation so that child classes not implementing it get a deprecation notice. 
It should then be used via `func_get_arg()` inside the method, until it's really added in 8.0